doc/Tutorials/Bokeh_Backend.ipynb

```diff
@@ -438,7 +438,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "macro_df = pd.read_csv('http://assets.holoviews.org/macro.csv', '\\t')"
+    "macro_df = pd.read_csv('https://assets.holoviews.org/macro.csv', '\\t')"
```

Comment:
Assume these will work fine but worth checking, don't think this is technically necessary.